templates/security/certifications.html

```diff
@@ -3,202 +3,203 @@
-        <li class="p-list__item"><a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/2888">OpenSSL validated level 09 Oct 2020 (#3725)</a></li>
-        <li class="p-list__item"><a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/2962">Kernel Crypto API validated level 07 Oct 2020 (#3724)</a></li>
-        <li class="p-list__item"><a class="p-link--external" href="https://csrc.nist.gov/projects/cryptographic-module-validation-program/Certificate/2978">Strongswan validated level 1 July 2017 (#2978)</a></li>
-      </ul>
-    </div>
-    <div class="col-4 u-hide--small u-align--center">
-      {{
-        image(
-          url="https://assets.ubuntu.com/v1/9f048bd7-NIST_logo.svg",
-          alt="",
-          height="58",
-          width="220",
-          hi_def=True,
-          loading="lazy",
-          attrs={"class": ""}
-        ) | safe
-      }}
-    </div>
-  </div>
+<section class="p-strip--light">
+ <div class="row p-divider">
+   <div class="col-4 p-divider__block">
+     <h3>Comply with established security baselines</h3>
+     <p>The default configuration of Ubuntu balances between usability and security. However, systems carrying dedicated workloads can be further hardened to reduce their attack surface. We work with DISA to ensure STIG guides available for Ubuntu, and we provide OpenSCAP tooling and automation for the Industry accepted CIS benchmark. Available with <a href="/advantage">Ubuntu Advantage</a> and <a href="/public-cloud">Ubuntu Pro</a>.</p>
+     <a class="p-button--neutral" href="#compliance-profiles">See our compliance profiles</a>
+   </div>
+   <div class="col-4 p-divider__block">
+     <h3>Transparency</h3>
+     <p>Canonical is transparent on the Security Features we enable on Ubuntu. We have a <a href="/security/notices">public vulnerability disclosure policy</a> and known vulnerabilities are disclosed in <a href="/security/oval">Machine readable OVAL</a> CVE output to be used by OpenSCAP and other 3rd party tools.</p>
+     <a class="p-button--neutral p-link--external" href="https://wiki.ubuntu.com/Security/Features">See our security features</a>
+   </div>
+   <div class="col-4 p-divider__block">
+     <h3>Security certification and 3rd party attestation</h3>
+     <p>Access to certification artifacts as well as the necessary tooling for regulated and high security environments. <a href="/advantage">Ubuntu Advantage</a> and Ubuntu Pro provide access to FIPS 140-2 certified cryptographic packages, allowing you to deploy workloads that need to operate under compliance regimes like FedRAMP, HIPAA, and PCI-DSS. Additionally, Ubuntu versions have been certified under Common Criteria, providing 3rd party attestation of the security mechanisms in the operating system.</p>
```

Comment:
Should 'Ubuntu Pro' also be a link here for consistency?

Reply:
Goos spot, makes sense!